ext/node/polyfills/internal_binding/handle_wrap.ts

```diff
@@ -40,7 +38,7 @@ export class HandleWrap extends AsyncWrap {
 
   close(cb: () => void = () => {}) {
     this._onClose();
-    queueMicrotask(cb);
+    setTimeout(cb, 0);
```

Comment:
Have you tried using `nextTick` instead?

Reply:
`nextTick` doesn't delay enough because `emitErrorNT` callback is called at next tick (and that's registered after `_handle.close` callback.

https://github.com/denoland/deno/blob/29934d558c188fdc3406706da19921ca5a389383/ext/node/polyfills/net.ts#L1418-L1426

In the above code, the callback for `this._handle.close` is the one delayed in this PR (that emits `close`). `cb` in the above is `onDestroy` in `_stream.mjs`, which calls `process.nextTick(() => stream.emit("error"))`.

So we need to delay the close callback at least 2 ticks.